__tests__/hooks/useFeedHeader.test.js

```diff
@@ -54,7 +54,7 @@ describe('Feed Header should run correctly', () => {
     act(() => {
       result.current.onBackNormalUser();
     });
-    expect(mockedNavigate).toHaveBeenCalled();
+    expect(mockedGoBack).toHaveBeenCalled();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `mockedNavigate` to `mockedGoBack` seems appropriate if the intention is to simulate a back navigation event. However, it's important to ensure that this change aligns with the actual behavior of the `onBackNormalUser` function. If the function is supposed to navigate to a specific screen rather than just going back, this test might not be accurate.
```diff
-    expect(mockedNavigate).toHaveBeenCalled();
+    expect(mockedGoBack).toHaveBeenCalled();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->